test/katex-spec.js

```diff
@@ -98,10 +98,12 @@ const parseAndSetResult = function(expr, result, settings) {
     } catch (e) {
         result.pass = false;
         if (e instanceof ParseError) {
-            result.message = "'" + expr + "' failed " +
+            result.message = () =>
```

Comment:
These changes seem unrelated to `addFontMetrics`.  Why the switch to a callback?

Reply:
I initially made a mistake in my code and some of tests failed but the error I got was `result.message is not a function`. I looked at the documentation of jest and found out that message should be a function.   
It seems like tests for these parts of `katex-spec` always passed so we never got any error that covered these error types so we never found out.

Reply:
I've encountered these bugs throughout as well. They are sometimes in my PRs, though I guess we should do a thorough cleanup.

Reply:
I think I've run into the same issue now that you mention it.  We should add `// @flow` to the test files at some point to (not now).